polyfills/Array/from/polyfill.js

```diff
@@ -1,3 +1,44 @@
+
+function parseIterable(arraylike) {
+	var
+	done = false,
+	iterableResponse,
+	tempArray;
```

Comment:
If we're importing from a third party repo, this would be fine, but since the polyfill service is to be the canonical home for this polyfill, the syntax should ahere to the jshint config provided, which includes no multi var declarations.